src/Utility/SecretGenerator.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Accompli\Utility;
+
+/**
+ * Generatator to generate secrets.
+ *
+ * @author Ron Rademaker
+ */
+class SecretGenerator implements ValueGeneratorInterface
```

Comment:
I think it's better to name this `ShaValueGenerator` instead of naming it to it's current usage.